examples/index.json

```diff
@@ -300,6 +314,27 @@
                 "verifyBuild": true,
                 "verifyOutput": true,
                 "isLearnByExample": true
+            },
+            {
+                "name": "Type inclusion for records",
+                "url": "type-inclusion-for-records",
+                "verifyBuild": true,
+                "verifyOutput": true,
+                "isLearnByExample": true
+            },
+            {
```

Comment:
IMO this is more relevant to the functions section rather than the records section.